packages/components/table/src/table-header/index.ts

```diff
@@ -67,13 +68,38 @@ export default defineComponent({
     const ns = useNamespace('table')
     const filterPanels = ref({})
     const { onColumnsChange, onScrollableChange } = useLayoutObserver(parent!)
+
+    const isTableLayoutAuto = parent?.props.tableLayout === 'auto'
+    const saveIndexSelection = reactive(new Map())
```

Comment:
```suggestion
    const saveIndexSelection = new Map()
```

IMO, the `reactive` are not necessary. WDYT

Reply:
Yeah, it seems unnecessary.